server/src/models/fake.database.ts

```diff
@@ -1,54 +1,81 @@
 import { User, IUserModel } from './user.model';
 import { Token, ITokenModel } from './token.model';
 import { Post, IPostModel } from './post.model';
+import { Comment, ICommentModel } from './comment.model';
 import { Like, ILikeModel } from './like.model';
 
 import '../helpers/globals';
 import connect from '../connect';
 import { hashPassword } from '../helpers/hash.password';
 
-import mongoose from 'mongoose';
+import mongoose, { Schema, Document, Model, model } from 'mongoose';
 import faker from 'faker/locale/en';
+import axios, { AxiosResponse } from 'axios';
+import FormData from 'form-data';
 
 faker.seed(983465);
 mongoose.set('useNewUrlParser', true);
 
-const { env: { DB_PATH } }: any = process;
+const { env: { DB_PATH, IMAGE_DB_PATH, IMGUR_CLIENT_ID } }: any = process;
 const FAKE_DB_SIZE = parseInt(process.env.FAKE_DB_SIZE, 10);
 
+const capitalizeSentence = function(sentence: string): string {
```

Comment:
Why do you use simple function instead of es6 arrow function?

Reply:
Fixed.